src/styles/courses.scss

```
@@ -0,0 +1,85 @@
+.courses {
+  padding-bottom: 120px;
+  background-color: #fff;
+  &__header {
+    margin: 0 auto;
+    font-size: 16px;
+    line-height: 19px;
+    text-align: center;
+    color: $color-thirty;
+    text-transform: uppercase;
+  }
+
+  &__items {
+    margin-top: 24px;
+    display: grid;
+    grid-template-columns: repeat(4, 1fr);
+
+    @media (max-width: $small-screen) {
+      display: block;
+    }
+
+    .courses__item {
+      position: relative;
+      height: 360px;
+      padding: 100px 20px 0 78px;
+      transition: transform 0.5s, border-radius 0.5s;
+
+      @media (max-width: $small-screen) {
+        padding-left: 0;
+        height: 250px;
+      }
+      .courses__list {
+        max-width: 255px;
+        font-size: 32px;
+        line-height: 34px;
+
+        @media (max-width: $small-screen) {
+          width: fit-content;
+          margin: 0 auto;
+        }
+      }
+      .courses__list::marker {
+        color: #e9b730;
+      }
+    }
+
+    .courses__item::before {
```

можно вложить в блок courses__item
```suggestion
    &::before {
```